modules/reliquary/components/RelicCarousel.tsx

```diff
@@ -29,8 +29,8 @@ export function RelicCarousel({ loading, ...rest }: Props) {
     return (
         <Box position="relative" minH="600px">
             {hasNoRelics && !_isLoadingRelicPositions && (
-                <Flex justifyContent="center" zIndex={2} width="full" position="absolute" left="0" right="0" top="10%">
-                    <VStack spacing="4" alignItems="center">
+                <Flex justifyContent="center" zIndex={2} width="full" alignItems="center" height="600px">
```

Comment:
maybe set the heights on line 30 & 32 to 300px so the button stays above the fold?